cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -475,6 +482,96 @@ private void updateConfigurationTable() {
   }
 
   /**
+   * The replication Status Tool coprocessors will use replication state table to track
+   * replication status. Create the table here.
+   */
+  private void createReplicationStateTable() throws IOException {
```

Comment:
we should add a config setting (that defaults to false) for whether replication is enabled. If not, we don't create the table or write to it.